src/components/AddCard/AddCard.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import { Modal, TextArea, TextInput } from "carbon-components-react";
+import { useUID } from "react-uid";
+
+const AddCard = (props) => {
+  const [cardHeaderValue, setCardHeaderValue] = useState("");
+  const [cardContentValue, setCardContentValue] = useState("");
```

Comment:
Better combine states, will allow you to use spread operator. And be free of changes if new fields are added to the card.